src/rqt_console/console_settings_dialog.py

```diff
@@ -32,11 +32,13 @@
 
 import os
 
+from ament_index_python.resources import get_resource
+
 from python_qt_binding import loadUi
 from python_qt_binding.QtWidgets import QDialog
 
-from rqt_logger_level.logger_level_widget import LoggerLevelWidget
-from rqt_logger_level.logger_level_service_caller import LoggerLevelServiceCaller
+# from rqt_logger_level.logger_level_widget import LoggerLevelWidget
```

Comment:
Mark as todo if this needs to be re-implemented

Reply:
done